pybamm/meshes/one_dimensional_submeshes.py

```diff
@@ -52,6 +52,9 @@ def near(x, point, tol=3e-16):
                         f"{tab} tab located at {tab_location}, "
                         f"but must be at either 0 or {l_z}"
                     )
+        elif tabs:
```

Comment:
so presumably "negative tab" will only be in `tabs` if the model has been serialised. I still don't understand why tho?

Reply:
See above - "negative tab" is present in `tabs` once a mesh has been initialised, but __init__ will only come across this if a mesh has been serialised and is now being re-created upon deserialisation.